scripts/publish/docs.js

```diff
@@ -2,25 +2,16 @@
 'use strict';
 
 /* eslint-disable no-console */
-const fs = require('fs-extra');
+const exec = require('child_process').exec;
+const fs = require('fs');
```

Comment:
Note that I changed `outputFile` to `writeFile`. Since there's no directories in the wiki they're identical.